### ISSUE TITLE
Allow everything to work with Parent/Staff

### ISSUE THREAD

**Issue by gormleycep** [cep-repo/YT-tools#10]
Started this in the testing-branch but more testing needs to be done. Consult with Josh and Sophie about how panels should be different form Family/Staff survey. It's actually a lot simpler. Will need to pull new field for Salesforce which I've updated in the testing-branch branch.

**Comment by cepbrian** [comment]
De we know what this ticket requires yet?

**Comment by sophiebyouthtruth** [comment]
We need F/S to work exactly like any ose/fft survey when using both YT tools and YDEA. I can identify the things that do not work the same when I make a F/S survey and report, does that help? Or is there a way to simply smoothly make F/S part of our process?

**Comment by cepbrian** [comment]
yeah, a list of things that don't work would be very useful. thanks!

**Comment by sophiebyouthtruth** [comment]
Ok. I can make a list as I go. So far, other than the "string v. int" issue we ran into with YDEA, a 2-0 does not work for F/S. This step takes qualtrics information from school_list.csv (example here: Dropbox (CEP)\YouthTruth\Report Production\_YDEA Archive\East Grand Forks\mde\es\combo) and knows to deactivate the Qualtrics survey and download the raw data in csv format to the /combo/raw folder.

**Comment by sophiebyouthtruth** [comment]
Next issue: When we run a 2-4, schoolinfo csv is created, but we need column comparative to =0, not 1. Or just be blank. 

Also in the schoolinfo csv, we need the columns "fam" "staff" and "state" to automatically come in. We have been manually adding these columns.

**Comment by sophiebyouthtruth** [comment]
Next issue: Keep getting error at a 2-5 when trying to upload schoolinfo. You say this error is because I'm trying to upload a float instead of an integer, but when I manually upload all the same info into schoolinfo in SQL, everything works fine. I haven't been doing anything differently. HEre's a photo of the error. An example of this happening is in Dropbox (CEP)\YouthTruth\Report Production\_YDEA Archive\_FamilyStaff\Penn-Delco School District Family\fs\ms

fs - ms - school_report at a 2-5. 

![image](https://cloud.githubusercontent.com/assets/12432159/14870035/389211e4-0c8c-11e6-8eac-b5a4eab1988d.png)

**Comment by sophiebyouthtruth** [comment]
Another issue (and I just realized this whole ticket is mixed between YDEA and YT Tools). This is a YDEA issue but related to fam/staff. When jsons are created and we put jsons in the ORS, the naming conventions are a little off.

Instead of "Mar 16" or "16-Mar" at the end of the report title, we need "Spring 2016." This is an easy manual fix for now.

The High School family survey is erroneously named a staff survey. As an example, I created a family json and inserted it into the family template and this is what it came out titled as:

Manor New Technology High School - "Staff Pilot Survey" School Report - Mar-16

it should read:
Manor New Technology High School - "Family Pilot Survey" School Report - Spring 2016.

Another quick manual fix, but will get confusing when we have more clients. Thanks!

**Comment by cepbrian** [comment]
I think that this is a separate issue. Please move this to YDEA.

I don't think that I can easily change the mar -> spring. It doesn't do this for OSE, does it? Why do we need spring instead of march? 

On the "staff" vs "family" isn't that part of the template? what variable does it get that from in the template?
